CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ Changelog
 
 ### Fixes
 
+- Fixed Wallet name length issues on smaller screen sizes ([PR 1689](https://github.com/input-output-hk/daedalus/pull/1689))
```

Comment:
@DeeJayElly this entry should go under new section "vNext" which you should add on top of the file like I did in my PR: https://github.com/input-output-hk/daedalus/pull/1690/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR4-R9

Reply:
Fixed